apps/fishing-map/features/map/Map.tsx

```diff
@@ -114,7 +112,7 @@ const MapWrapper = () => {
         id={MAP_CANVAS_ID}
         ref={deckRef}
         views={MAP_VIEW}
-        layers={deckRef ? (layers as LayersList) : []}
+        layers={deckRef ? ([...layers, ...overlays] as LayersList) : []}
```

Comment:
we might skip some layers rendering if we wrap this layers within a useMemo, don't we?